src/log4net/Appender/RollingFileAppender.cs

```diff
@@ -681,18 +681,16 @@ protected List<string> GetExistingFiles(string baseFilePath)
     using (SecurityContext?.Impersonate(this))
     {
       string fullPath = Path.GetFullPath(baseFilePath);
-      string dir = Path.GetDirectoryName(baseFilePath);
 
       directory = Path.GetDirectoryName(fullPath);
       if (Directory.Exists(directory))
       {
         string baseFileName = Path.GetFileName(fullPath);
-
+        
```

Comment:
You might want to clean that whitespace change.

Reply:
Will do, thanks!